CHANGELOG.md

```
@@ -1,3 +1,18 @@
+## [9.29.0] - 2026-04-22
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)